.bazelci/examples.yml

```diff
@@ -22,6 +22,10 @@ tasks:
     name: "Android instrumentation test example"
     platform: ubuntu1804
     working_directory: examples/android_instrumentation_test
+    test_flags:
+      # Compatibility with --incompatible_use_python_toolchains
+      # https://github.com/bazelbuild/bazel/issues/7899
+      - "--host_force_python=PY2"
```

Comment:
Shall we put it in a .bazelrc file?

Reply:
I'm upstreaming a change to the broken dependency.